CalculatorW.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CalculatorW.Tests;
+using CalculatorWindowsForms;
```

Comment:
```suggestion
﻿namespace CalculatorW.Tests;
﻿
using CalculatorWindowsForms;
```
И нужна шапка с лицензией и более говорящее имя файла